dataplug-instagram/conf/evolutions/dataplug.sql

```diff
@@ -7,3 +7,9 @@ VALUES
   ('feed', 'User feed''s media', 'sequence'),
   ('profile', 'User''s Instagram profile information', 'snapshots')
   ON CONFLICT (name) DO NOTHING;
+
+--changeset dataplugInstagram:deleteEndpoints context:data
```

Comment:
Can we archive all of these endpoints into a separate table before deleting?

Reply:
Let's follow a convention of naming archive tables with `_archive` suffix. Please change `dataplug_user_old` -> `dataplug_user_archive`.